pom.xml

```diff
@@ -155,6 +155,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-spark-runtime-3.3_2.12</artifactId>
+            <version>0.14.0</version>
+            <scope>test</scope>
+        </dependency>
 
```

Comment:
Correct me if I'm wrong, but we don't seem to be using any specific feature of Iceberg. If that's the case, can we remove this dependency ?

Reply:
in tests I have added, test to check if Iceberg table writes are working as expected. So that's the reason to add this dep in test scope.